<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomUi</class>
 <widget class="QWidget" name="RoomUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_camera_open">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OPEN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_camera_close">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_roommate">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>80</y>
     <width>161</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_roommate">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ROOMMATES:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_flash">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>FLASH</string>
   </property>
  </widget>
  <widget class="QLabel" name="lable_camera">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>VIDEO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_make">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MIKE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_make_open">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>230</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OPEN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_make_close">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>280</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>671</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>671</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>590</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
  </widget>
  <widget class="VideoLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>551</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_danmu">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>320</y>
     <width>63</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>DANMU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_danmu">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>350</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>420</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>GIFTS</string>
   </property>
   <widget class="QCheckBox" name="checkBox_runcar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>RUNCAR</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_diamond">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>DIAMOND</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_flower">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>FLOWER</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoLabel</class>
   <extends>QLabel</extends>
   <header>VideoLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
